content/en/security/account_takeover_protection.md

```diff
@@ -0,0 +1,268 @@
+
+### Know your trends
+
+Use the [Threats Overview][11] to monitor business logic trends, such as spikes in failed logins against your services.
+
+{{<img src="security/ato/threats_overview2.png" alt="Threats Overview" style="width:100%;">}}
+
+
+### Signal review
+
+Review the following best practices for signals.
+
+#### IP addresses
+
+Use short durations for blocking attackers. 15 minutes or less is recommended. It is uncommon for attackers to reuse IP addresses in distributed account takeovers.
+
+#### Data centers
+
+Some attacks are launched using inexpensive virtual private servers (VPS) and hosting providers. Attackers are motivated by how their low cost and automation let them access new IP addresses at the data center.
+
+Many consumer applications have low occurrences of user authentication from data centers, especially low cost data centers and VPS providers. Consider blocking the entire data center or ASN when the network range is small, and not within your expected user authentication behavior.
+
+<div class="alert alert-info">Datadog uses third party data sources such as IPinfo and Spur to determine if an IP is a hosting provider. Datadog processes this data within Datadog infrastructure.</div>
+
+#### Proxies
+
+Datadog uses [Spur][8] to determine if an IP is a proxy. Datadog correlates indicators of compromise (IOCs) with account takeover attacks for faster detection with the ASM-managed account takeover rules.
+
+Datadog recommends never blocking IP addresses solely based on threat intelligence IOCs for IP addresses. See our threat intelligence [best practices][9] for details.
+
+Details on IPs, including ownership and threat intelligence, are available in the IP address details. Click on an IP addresses to view these details.
+
+There are two types of proxies frequently seen in distributed account takeovers:
+
+- Hosting proxies: These are proxies that exist at data centers, and are often the result of a compromised host at that data center. Guidance for interacting with hosting proxies is similar to data centers.
+
+- Residential proxies: These are proxies that exist behind residential IPs. Residential proxies are frequently enabled by mobile application SDKs or browser plugins. The user of the SDK or plugin is typically unaware that they are running a proxy. It is common to see benign traffic from IP addresses identified as residential proxies.
+
+#### Mobile ISPs
+
+Datadog uses third parties such as IPinfo and Spur to determine if an IP is a Mobile ISP.
+
+Exercise caution when blocking Mobile ISPs. Mobile ISPs use [CGNAT][10] and frequently have large numbers of phones behind each IP address. 
+
+#### Attacker attributes
+
+Use attacker attributes to target response actions.
+
+Datadog clusters attackers using attacker similarity and identifies the attributes of those attackers. Responders can use custom rules to block the attributes of persistent attackers.
```

Comment:
```suggestion
Datadog clusters attackers by the similarity of their attributes. Responders can use custom rules to block the attributes of persistent attackers.
```
Or similar if I've mangled the meaning, I just found the sentence as written a little tough to parse out